docs/static/releasenotes.asciidoc

```diff
@@ -76,12 +76,43 @@ This section summarizes the changes in the following releases:
 === Logstash 8.17.0 Release Notes
 
 [[known-issues-8-17-0]]
-==== Known issue
+==== Known issues
+
+[[known-issue-8-17-0-jvm]]
+===== JVM version changes needed when upgrading Logstash from 8.12.0 (or earlier)
+
+If the `jvm.options` file was modified and not overwritten with the newest version, you may see a "deserialize invocation error" message, causing the pipeline to crash. 
+Users are affected if the Persistent Queue (PQ) is enabled, and the pipeline is processing messages larger than 20MB.
+
+**Solution:** Apply the default change contained in the newer 'jvm.options' file, as seen in this https://github.com/elastic/logstash/blob/main/config/jvm.options#L66-L82[example].
+
+[[known-issue-8-17-0-json_lines]]
+===== Error with `json_lines` codec 3.2.0 (or earlier) and {ls} 8.17.1 (or earlier)
```

Comment:
```suggestion
===== Error with `json_lines` codec 3.2.0 (or earlier) with {ls} 8.17.0
```
`and {ls} 8.17.1 (or earlier)` - part is a bit confusing to me. I support removing LS version from this title which brings reader to "... present in {ls} versions `8.16.0`, `8.16.1`, and `8.17.0`." and to mitigation section.

Reply:
This was based on description in issue: "When using the json_lines codec > 3.2.0 with versions of **_Logstash < 8.17.1_**, the following error may be emitted, crashing the pipelines"

If that's not accurate, I'll update it here, and it will also need to be updated in the issue.

Reply:
What do you think about using the _symptom_ as the heading, rather than `json_lines` codec?

```
===== "Input Buffer Full" error With Logstash `8.17.0`, `8.16.1` and `8.16.0`
```

Then describe the instances of how this would happen and how to mitigate, with Logstash->Logstash communication first, as it is the most likely, then the  `json_lines` case?

Leading with `json_lines` isn't necessarily obvious as the codec is used "silently" by the Logstash integration plugin.